server/src/test/java/com/octopus/teamcity/opentelemetry/server/TestOTELHelperFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.octopus.teamcity.opentelemetry.server;
```

Comment:
Are these tests adding anything valuable? Or are they scaffolding for future?

Reply:
These aren't tests, these are a mock factory to use for the existing tests